### ISSUE TITLE
High loss curves, what's wrong? Some help please 

### ISSUE THREAD

**Issue by Amrimn** [matterport/Mask_RCNN#1904]
Hello @waleedka,
I am trying to use this code to build an object detection model with gray level satellite images.
I am training on a data set(~9000) of 512 gray level satellite images.
The global validation loss is hight >2 however the mask, Bbox and class loss are not that hight?
I'm suffering from overfitting, what can I do to make it better?

**Configuration:**
GPU_COUNT                      1
GRADIENT_CLIP_NORM             5.0
IMAGES_PER_GPU                 8
IMAGE_MAX_DIM                  512
IMAGE_META_SIZE                14
IMAGE_MIN_DIM                  512
IMAGE_MIN_SCALE                0
IMAGE_PADDING                  True
IMAGE_RESIZE_MODE              none
IMAGE_SHAPE                    [512 512   3]
LEARNING_MOMENTUM              0.9
LEARNING_RATE                  0.001
LOSS_WEIGHTS                   {'rpn_class_loss': 1.0, 'rpn_bbox_loss': 1.0, 'mrcnn_class_loss': 1.0, 'mrcnn_bbox_loss': 1.0, 'mrcnn_mask_loss': 1.0}
MASK_POOL_SIZE                 14
MASK_SHAPE                     [28, 28]
MAX_GT_INSTANCES               200
MEAN_PIXEL                     [123.7 116.8 103.9]
MINI_MASK_SHAPE                (56, 56)

![1](https://user-images.githubusercontent.com/38752643/70613632-42d8a880-1c09-11ea-80ba-348c954e4826.png)
![2](https://user-images.githubusercontent.com/38752643/70613629-42401200-1c09-11ea-9000-49420bfb4311.png)
![3](https://user-images.githubusercontent.com/38752643/70613630-42401200-1c09-11ea-8d97-aa7687f7b919.png)
![5](https://user-images.githubusercontent.com/38752643/70613631-42d8a880-1c09-11ea-8084-575a71f986ba.png)

Can you please help me fix this? Thanks in advance.

**Comment by akrsrivastava** [comment]
Where are your rpn_bbox_loss and rpn_class_loss graphs? Check them. Global Loss is the sum of individual losses

**Comment by Amrimn** [comment]
@akrsrivastava  here they are. 
![Capture0](https://user-images.githubusercontent.com/38752643/70982354-24662780-20b7-11ea-95a6-d1b6f0729a37.PNG)

**Comment by akrsrivastava** [comment]
Sorry I meant the val_rpn_bbox_loss and val_rpn_class loss
Although, I checked your training losses. The sum of the losses seem to be roughly equal to the "loss"

**Comment by Amrimn** [comment]
Here is the loss smoothed
![Capture0](https://user-images.githubusercontent.com/38752643/70986251-d43e9380-20bd-11ea-94dd-a744a7ca56ea.PNG)

**Comment by akrsrivastava** [comment]
Clearly, the sum of all validation losses is equal to val_loss. I dont see any issue here.

**Comment by Amrimn** [comment]
@akrsrivastava the probem is that my loss is very high and its due to the val_rpn_bbox_loss i think.
What can i do to make the validation loss less and more smooth?

**Comment by geobowen** [comment]
@Amrimn Excuse me, I happend to have the similar problem. Did you solve it now? I utilized a 2 stage training strategy. The first stage is to train the head layers with a lr=0.002 for 2 epoches and for all layers with lr=0.001 for 40 epoches. From the loss graph i noticed that after the head layers training is done, the val loss especially the val_rpn_bbox goes pretty high and it reaches  5 or 6 or even higher. Do you have any idea about what may go wrong in this process.
Best Regards
